frontend/next.config.js

```diff
@@ -29,7 +29,7 @@ const removeImports = require("next-remove-imports")({
 })
 const nextTranslate = require("next-translate")
 
-const mediaUrl = new URL(process.env.MEDIA_URL ?? "http://localhost")
+const mediaUrl = new URL(process.env.MEDIA_URL != null ? process.env.MEDIA_URL : "http://localhost")
```

Comment:
Can you explain the reasoning for this change?

Reply:
For some reason, the ``??`` won't mesh well with the latest javascript install on my machine.

Reply:
That's weird, what's your `node --version`?

Reply:
v12.22.9

Reply:
Hm, `??` should be supported OK there. By "won't mesh well" what exactly do you mean? Did it complain of a syntax error?

Reply:
Yes.

Reply:
Updated node, removing the need for this workaround. Thus, I reverted the workaround.